app/src/org/commcare/network/RemoteDataPullResponse.java

```diff
@@ -128,4 +129,8 @@ protected long guessDataSize() {
         }
         return -1;
     }
+
+    public String getRetryHeaderValue() {
+        return response.getFirstHeader("Retry-After").getValue();
```

Comment:
Interesting question.. it feels like yes because it means HQ just totally messed up the response, but I could see an argument for doing something better..